jest.setup.js

```diff
@@ -3,17 +3,24 @@ import { directiveSsr as t } from '@shell/plugins/i18n';
 import VTooltip from 'v-tooltip';
 import VModal from 'vue-js-modal';
 import vSelect from 'vue-select';
+import ClientOnly from 'vue-client-only';
 
 import Vue from 'vue';
 
 Vue.config.productionTip = false;
 Vue.use(VTooltip).use(VModal);
 Vue.component('v-select', vSelect);
 
+// Component: <ClientOnly> (originally from Nuxt)
+Vue.component(ClientOnly.name, ClientOnly);
+
+Vue.component('codemirror', () => {});
```

Comment:
Code mirror issues have to be handled within it's own test.

Reply:
Its a global component that is normally loaded by the code mirror provider - if we add the component in the keyvalue test, it will then be available globally, so it seems to make more sense to add it globally - which is effectively what we do in the app.

Under your argument, v-select should not be added here either.